modules/user0_webauth/user0_webauth.info

```diff
@@ -0,0 +1,5 @@
+name = Webauth for anonymous Drupal user

```

Comment:
This module will not work in Acquia, and we are aiming to shut down the on-premises infrastructure (and WebAuth) by the end of the year.